extensions/markdown-language-features/src/util/topmostLineMonitor.ts

```diff
@@ -22,6 +30,21 @@ export class TopmostLineMonitor extends Disposable {
 				}
 			}
 		}));
+
+		this._register(vscode.window.onDidChangeActiveTextEditor(textEditor => {
+
+			// When at a markdown file, apply existing scroll settings from static preview if applicable.
+			// Also save reference to text editor for line number reference later
+			if (textEditor && isMarkdownFile(textEditor.document!)) {
+
+				if (this.previousStaticEditorInfo.uri?.toString() === textEditor.document.uri.toString()) {
+					const line = this.previousStaticEditorInfo.line ? this.previousStaticEditorInfo.line : 0;
+					scrollEditorToLine(line, textEditor);
```

Comment:
Should the TopMostLineMonitor being scrolling editors or can this live somewhere else?

Reply:
Moved to `MarkdownPreviewManager`